src/features/application/components/Apply.js

```diff
@@ -91,30 +138,51 @@ function Apply() {
     formik.setFieldValue("jobPosition", props.value);
   };
 
+  const handleOnCountryChange = (props) => {
+    formik.setFieldValue("country", props.value);
+    const country = countryInfo.find((country) => country.name === props.label);
+    setCallingCode(country.callingCodes[0]);
+    setFlag(country.flag);
+  };
+
+  React.useEffect(() => {
+    const getCountries = async () => {
+      const result = await fetch(process.env.REACT_APP_COUNTRIES);
+      const data = await result.json();
+      data.map((country) =>
+        countries.push({ value: country.alpha3Code, label: country.name })
```

Comment:
- I don't understand this part of the code. You are pushing into array `countries` but you are not setting the state, you are not using the function `setCountries`. And after that, you are using the `countries` array.

Reply:
My bad, this time i used setCountries